lib/java/com/google/android/material/button/MaterialButtonHelper.java

```diff
@@ -46,11 +45,15 @@
 import com.google.android.material.shape.ShapeAppearanceModel;
 import com.google.android.material.shape.Shapeable;
 
+import static androidx.annotation.RestrictTo.Scope.LIBRARY_GROUP;
+
 /** @hide */
 @RestrictTo(LIBRARY_GROUP)
 class MaterialButtonHelper {
 
-  private static final boolean IS_LOLLIPOP = VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP;
+  private static final boolean IS_MIN_LOLLIPOP = VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP;
```

Comment:
IS_LOLLIPOP_OR_ABOVE?

Reply:
Good point, wasn't "IS_MIN" and old used convention in Android when gating min API logic? I'm pretty sure that I saw some time ago code using this prefix.

Reply:
I guess Android uses atLeastXXXX in many places, which also works. : )